openupgrade_scripts/scripts/analytic/16.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,83 @@
+from openupgradelib import openupgrade
+
+_fields_renames = [
+    (
+        "account.analytic.account",
+        "account_analytic_account",
+        "group_id",
+        "plan_id",
+    ),
+    (
+        "account.analytic.line",
+        "account_analytic_line",
+        "group_id",
+        "plan_id",
+    ),
+]
+_models_renames = [("account.analytic.group", "account.analytic.plan")]
+_tables_renames = [("account_analytic_group", "account_analytic_plan")]
+
+
+def analytic_account_set_group_id_if_null(env):
+    """
+    We would like to fill for group_id column before start renaming it into plan_id
+    """
+    openupgrade.logged_query(
```

Comment:
I think in production database it is quite rare to have `analytic_account` with empty `analytic_group` right?